### PR TITLE
Update cgmanifest with some SPECS-EXTENDED entries

### PR DESCRIPTION
**Pull request by MateuszMalisz**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Adding entries to cgmanifest for some packages that do not have them yet, from SPECS-EXTENDED.

###### Change Log  

- Add entries to cgmanifest for packages rpm-mpi-hooks, sblim-cmpi-syslog, servletapi4, servletapi5, setuptool, sgpio, splix, ttmkfdir, virt-who, wsmancli, xhtml1-dtds, xmldb-api, xrestop

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- N/A

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -22440,6 +22440,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "rpm-mpi-hooks",
+          "version": "6",
+          "downloadUrl": "https://src.fedoraproject.org/rpms/rpm-mpi-hooks/blob/rawhide/f/mpi.attr"
```
Comment (PawelWMS) [inline cgmanifest.json:22449]: I forgot to mention, that we also need to update the spec files themselves, so they contain the correct URLs. Passing the GitHub checks for these specs is a good indicator, if we've done it right or not.

Comment (PawelWMS) [inline cgmanifest.json:22449]: For this one we'd need to use this:
```suggestion
          "downloadUrl": "https://src.fedoraproject.org/rpms/rpm-mpi-hooks/raw/rawhide/f/mpi.attr"
```

We want only the file, not the whole web page.

**Review by PawelWMS — CHANGES_REQUESTED** [review]
One comment about the links, looks good in general, we just need to also update the spec files.

File: SPECS-EXTENDED/wsmancli/wsmancli.spec [hunk 2]
```diff
@@ -2,12 +2,10 @@ Vendor:         Microsoft Corporation
...
 Name:           wsmancli
 Version:        2.6.0
-Release:        12%{?dist}
+Release:        13%{?dist}
 License:        BSD
 Url:            http://www.openwsman.org/
-# You can get this tarball here:
-# https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz
-Source:         wsmancli-%{version}.tar.gz
+Source0:        https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz
```
Comment (PawelWMS) [inline SPECS-EXTENDED/wsmancli/wsmancli.spec:8]: You accidentally changed the source tarball's name:

```suggestion
Source0:        https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

File: SPECS-EXTENDED/xhtml1-dtds/xhtml1-dtds.spec [hunk 3]
```diff
@@ -1,18 +1,17 @@
 %global date    20020801
 
+# W3C Software License for DTDs etc:
+# http://www.w3.org/Consortium/Legal/IPR-FAQ-20000620#DTD
 Name:           xhtml1-dtds
 Version:        1.0
-Release:        20020802%{?dist}
+Release:        20020803%{?dist}
```
Comment (PawelWMS) [inline SPECS-EXTENDED/xhtml1-dtds/xhtml1-dtds.spec:7]: Ah, now I see what you meant about the release and date for this spec. For some reason I thought the version was the date incremented by one. Yeah, this is not great but fixing this is still out of scope for this PR.

File: SPECS-EXTENDED/xmldb-api/xmldb-api.spec [hunk 4]
```diff
@@ -22,14 +22,13 @@ Distribution:   Mariner
 %define cvs_version 20041010
 Name:           xmldb-api
 Version:        0.1
-Release:        27%{?dist}
+Release:        28%{?dist}
 Summary:        XML:DB API for Java
 License:        Apache-1.1
-Group:          Development/Libraries/Java
```
Comment (PawelWMS) [inline SPECS-EXTENDED/xmldb-api/xmldb-api.spec:28]: I think you accidentally removed the `Group` tag.

Comment (MateuszMalisz) [reply SPECS-EXTENDED/xmldb-api/xmldb-api.spec:28]: I actually removed it on purpose. AFAIK Group tags serve no purpose and are not standardized in any way, so I figured removing that field would clean the SPEC up a bit.

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**